CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ if(${CMAKE_EXTRA_GENERATOR} MATCHES "Eclipse CDT4")
     set(CMAKE_ECLIPSE_VERSION "4.6.2" CACHE STRING "Eclipse version" FORCE)
 endif()
 
+string(REGEX REPLACE "-rc" "." DEP_VERSION ${DEP_VERSION})
```

Comment:
But if you have `VERSION=2.8.6-alpha`, then you get this output:
```
[0/1] Re-running CMake...
Dependencies: PACKAGE_NAME=realm-core;VERSION=2.8.6-alpha
CMake Error at CMakeLists.txt:20 (project):
  VERSION "2.8.6-alpha" format invalid.


-- Configuring incomplete, errors occurred!
```
Isn't that fine?